packages/@react-aria/radio/src/useRadioGroup.ts

```diff
@@ -44,11 +60,18 @@ export function useRadioGroup(props: RadioGroupProps, state: RadioGroupState): R
   });
 
   return {
-    radioGroupProps: {
+    radioGroupProps: mergeProps(domProps, {
+      // https://www.w3.org/TR/wai-aria-1.2/#radiogroup
```

Comment:
does this link end up in the documentation at all? i can't find it, there's the link at the top of the documentation, but it's different than this one

Reply:
nah, it's just a comment